front/src/modules/object-metadata/hooks/useFindManyObjectMetadataItems.ts

```diff
@@ -62,8 +63,14 @@ export const useFindManyObjectMetadataItems = ({
             .getLoadable(objectMetadataItemsState)
             .getValue();
 
-          if (!isDeeplyEqual(objectMetadataItems, actualObjectMetadataItems)) {
+          const isMockMode = snapshot.getLoadable(isMockModeState).getValue();
+
+          if (
+            isMockMode ||
```

Comment:
I feel this logic is a bit weird. why is the findMany function in charge of setting isMockModeState back to false. I feel it should be triggered by the business part in charge of it (sign-in / sign-up)